runtime/parachains/src/scheduler.rs

```diff
@@ -495,6 +495,7 @@ impl<T: Trait> Module<T> {
 
 		Scheduled::set(scheduled);
 		ParathreadQueue::set(parathread_queue);
+		AvailabilityCores::set(cores);
```

Comment:
😄

Reply:
Yeah, in the end everything was there :D